test/dynamics/webApi/create.test.js

```diff
@@ -31,38 +31,30 @@ describe('Dynamics - create', async () => {
     it('should build correct request using required parameters', async () => {
       const { createEnrolment, getToken, getMappings } = idm.dynamics
 
-      const { enrolmentStatus, enrolmentType } = getMappings()
+      const { enrolmentStatus } = getMappings()
       const token = await getToken()
 
-      const request = await createEnrolment.buildRequest('a20e6efe-9954-4c5b-a76c-83a5518a1385', 'b20e6efe-9954-4c5b-a76c-83a5518a1385', null, 'd20e6efe-9954-4c5b-a76c-83a5518a1385', enrolmentStatus.pending, enrolmentType.other)
+      const request = await createEnrolment.buildRequest('a20e6efe-9954-4c5b-a76c-83a5518a1385', 'b20e6efe-9954-4c5b-a76c-83a5518a1385', enrolmentStatus.pending, null, 'd20e6efe-9954-4c5b-a76c-83a5518a1381', 'd20e6efe-9954-4c5b-a76c-83a5518a1382', true)
 
       const expectedRequestObj = {
         'method': 'POST',
-        'url': `${dynamicsRoot}/defra_createenrolment`,
+        'url': `${dynamicsRoot}/defra_lobserviceuserlinks`,
         'headers': {
           'Authorization': `Bearer ${token}`,
           'Accept': 'application/json',
           'Cache-Control': 'no-cache',
           'Content-Type': 'application/json; charset=utf-8',
           'OData-MaxVersion': '4.0',
           'OData-Version': '4.0',
-          'Prefer': 'odata.maxpagesize=500, odata.include-annotations="*"'
+          'Prefer': 'return=representation'
```

Comment:
Are we worried about page size at all in the future?

Reply:
We are, but this header change is only for the create - so don't need to worry about it in this instance